food-bank-app/Main.tsx

```diff
@@ -2,11 +2,8 @@ import { NavigationContainer } from '@react-navigation/native'
 import { ThemeProvider, createTheme } from '@rneui/themed'
 import { createNativeStackNavigator } from '@react-navigation/native-stack'
 import { useAuthContext } from './src/contexts/AuthContext'
-import HomeScreen from './src/screens/HomeScreen'
-import Login from './src/screens/Login/Login'
-import StartSignup from './src/screens/Signup/StartSignup'
-import PersonalSignup from './src/screens/Signup/PersonalSignup'
-import RFCSignup from './src/screens/Signup/RFCSignup'
+
+import { HomeScreen, Donation, Login, SignupStart, SignupPersonal, SignupRFC } from './src/screens'
```

Comment:
```suggestion
import {
  HomeScreen,
  Donation,
  Login,
  SignupStart,
  SignupPersonal,
  SignupRFC
 } from './src/screens'
```

Pa dejarlo un poco más limpio, más adelante podríamos pasarlo a nuevas líneas